Misc/NEWS.d/next/Library/2024-09-16-12-31-48.gh-issue-123978.z3smEu.rst

```diff
@@ -0,0 +1 @@
+Remove broken :func:`time.thread_time` on NetBSD.
```

Comment:
```suggestion
Remove broken :func:`time.thread_time` and :func:`time.thread_time_ns` on NetBSD.
```
Also we are [removing](https://github.com/python/cpython/blob/a15a584bf3f94ea11ab9363548c8872251364000/Modules/timemodule.c#L1895) the `thread_time_ns` function.